lib/engine/game/g_18_norway/steps/nationalize_corporation.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18Norway
+      module Step
+        class NationalizeCorporation < Engine::Step::Base
+          def description
+            'Nationalization offer'
+          end
+
+          def actions(entity)
+            return [] if !entity.corporation? || entity != current_entity
+            return [] if @game.nationalized?(entity.corporation)
+
+            actions = []
+            actions << 'choose'
+            actions
+          end
+
+          def choice_name
+            'Nationalize Croporation'
+          end
+
+          def log_skip(entity)
+            return if @game.nationalized?(entity.corporation)
+          end
```

Comment:
This method is never going to do anything, which is probably correct. But it can be simplified to:
```suggestion
          def log_skip(_entity); end
```

Reply:
Added missing super